add_organism.py

```diff
@@ -0,0 +1,141 @@
+
+import sys
+import pprint 
+import os
+import pickle
+import subprocess
+import json
+from pprint import pprint
+
+os.environ.setdefault('DJANGO_SETTINGS_MODULE', 'phageAPI.settings')
+import django
+django.setup()
+
+from restapi.models import (
+    Organism,
+    Spacer,
+    Repeat,
+    LocusSpacerRepeat,
+    AntiCRISPR,
+    Locus
+)
+
+### Constants ###
+Entrez_email = 'wenxinchen11@hotmail.com'
+CRISPRFinder_output_folder = 'output_files'
+
+def pickle_item(item, filename):
+
+    with open(filename, 'wb') as handle:
+        pickle.dump(item, handle, protocol=pickle.HIGHEST_PROTOCOL)
+
+def get_pickle(filename):
+
+    f = None
+
+    if os.path.isfile(filename):
+        with open(filename, 'rb') as handle:
+            f = pickle.load(handle)
+    return f
+
+def add_sequence_to_db(accession, name):
+
+    o, created = Organism.objects.get_or_create(
+        name=name, accession=accession)
+    
+    print(o)
+    # Avoid Duplicate Entries (Comment out for testing)
+    if not created:
+        return created
+
+    # == for testing only ==
+    seq = get_pickle(accession)
+    if not seq:
+    # == for testing only ==
+        seq = get_ncbi_seq(Entrez_email, 'nuccore', 'fasta', accession)
```

Comment:
I'm getting an error: `name 'Entrez' is not defined` when I try this with a different accession (NC_015138)

Reply:
Forgot to import Entrez, fixed